app/templates/suppliers/edit_company_registration_number.html

```diff
@@ -20,61 +22,88 @@
         "href": url_for('.supplier_details')
       },
       {
-        "text": "Company registration number"
+        "text": page_name,
       }
     ]
   }) }}
 {% endblock %}
 
 {% block mainContent %}
 
-<div class="single-question-page">
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l">Are you registered with Companies House?</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form method="POST" action="{{ url_for('.edit_supplier_registration_number') }}" novalidate>
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+
+      {% set companiesHouseNumberHtml %}
+      {{ govukInput({
+        "id": form.companies_house_number.id,
+        "name": form.companies_house_number.name,
+        "label": {
+          "text": "Companies House number"
+        },
+        "hint": {
+          "html": "<a class='govuk-link' href='https://beta.companieshouse.gov.uk/search/companies'>Find your 8 digit Companies House number</a>"|safe,
```

Comment:
I realise this is a new feature, but I think this link should probably open in a new tab

Reply:
I'll put that in a tiny follow-up PR